tests/test_helpdesk.py

```diff
@@ -0,0 +1,159 @@
+
+from django.conf import settings
+
+from pytest import mark
+
+# from conversations.models import Message
+from conference.models import Conference
+from conversations.common_actions import (
+    mark_thread_as_completed,
+    staff_reply_to_thread,
+    user_reply_to_thread,
+)
+from conversations.helpdesk.api import (
+    get_actionable_helpdesk_threads,
+    create_new_support_request,
+)
+from conversations.models import Thread
+
+from .common_tools import sequence_equals, make_user
+
+
+def make_conference():
+    return Conference.objects.create(
+        code=settings.CONFERENCE_CONFERENCE,
+        name=settings.CONFERENCE_CONFERENCE,
+    )
+
+
+@mark.django_db
```

Comment:
In the case like this file, where every test is marked with `django_db`, you can declare `pytestmark = pytest.mark.django_db` in the global scope - this marks each test with `django_db` for you.

Reply:
I wrote it a long time ago, but in the mean time I noticed that we can just pass the `db` fixture, and it looks like a cleaner solution, so I'll just migrate all of those to use the fixture

Reply:
btw – thanks for the tip – TIL.